doc/BEGINNERS_TUTORIAL.org

```diff
@@ -0,0 +1,184 @@
+* Why Spacemacs?
+- Unparallelled text and structure editing for all types of writing tasks: creative writing, blogging, note-taking, todo-lists,
+  scientific papers... 
+- Powerful modes for programming in dozens of programming languages
+- Deeply customizable yet beginner-friendly
+* Installation
+ Spacemacs is a beginner-friendly and powerful distribution of a popular text
+ editor called Emacs. To install Spacemacs you need to first install base Emacs and then download
+ and install the Spacemacs extension files, which is easiest done by using a program
+ called Git. The steps are easy and outlined below.
+** 1. Install Emacs
+Follow the instructions in https://github.com/syl20bnr/spacemacs#prerequisites.
+** 2. Install Git
+Go to https://git-scm.com/downloads and install Git on your platform.
+** 3. Install Spacemacs
+Open a terminal or command prompt, paste the following code to it:
+
+git clone https://github.com/syl20bnr/spacemacs ~/.emacs.d
```

Comment:
Ah, I don't know how it works on Windows (never really used it oO).

My point was about document structure :)